src/test/java/org/terasology/flexiblepathfinding/FallingJPSTest.java

```diff
@@ -0,0 +1,184 @@
+// Copyright 2022 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.flexiblepathfinding;
+
+import org.junit.jupiter.api.Test;
+import org.terasology.flexiblepathfinding.helpers.JPSTestHelper;
+import org.terasology.flexiblepathfinding.helpers.TestPaths;
+import org.terasology.flexiblepathfinding.helpers.TestWorlds;
+import org.terasology.flexiblepathfinding.plugins.basic.FallingPlugin;
+
+/**
+ * Falling movements are purely vertical (down) and require additional horizontal walking movements
+ * The following test cases only attempt to test that the falling plugin alone does not incorrectly find valid paths for basic movements
```

Comment:
Can you add one positive test case to ensure that as well? Basically, finding a path from a position in the air one block down to a "grounded" block.

Reply:
done